components/style/binding_tools/regen.py

```diff
@@ -106,6 +106,7 @@
             "AnonymousContent",
             "AudioContext",
             "CapturingContentInfo",
+            "ConsumeStyleBehavior",
```

Comment:
These should be noops, right? They're in the mozilla namespace above. Anyway, happy to clean it up afterwards.